docs/installation/local.md

```diff
@@ -0,0 +1,26 @@
+# Local
+Follow these steps to set up the project locally.
+## Dependencies
+You will need the following things properly installed on your computer.
+
+* **[Git](https://git-scm.com/)**
+* **[Node.js 8.x LTS](https://nodejs.org/)**
+* **[Yarn](https://yarnpkg.com/en/docs/install)**
+* **[Ember CLI](https://ember-cli.com/)** - `yarn global add ember-cli`
+
+It is also recommended to have [watchman](https://facebook.github.io/watchman/docs/install.html) installed to speed up the `file-watcher/auto-build` service of the ember build server.
+
+## Steps
+* `git clone <repository-url>` this repository
+* `cd open-event-frontend`
+* `yarn`
+* `cp .env.example .env`
+
+### Note:
+If you face error in `npm install` for node and npm version, then install node version: v7.10.1 (npm version: 5.3.0 gets installed with it), which satisfies check-node-version and project runs perfectly on this configuration.
```

Comment:
This can be removed. As we no longer recommend npm. This project comes bundled only with a yarn-compatible lockfile.